### PR TITLE
Admin ui

### PR DESCRIPTION
**Pull request by WelltokCaleb**

### CODE REVIEW

File: fields/types/number/NumberValidation.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	ValidateNumber: function (value, rules, field) {
+
+		if (rules.minValue !== undefined) {
+			if (value < rules.minValue) {
+				field.options.invalidMessage = 'The number in the ' + field.label + ' field must be greater than or equal to' + rules.minValue + '.';
+				return false;
+			}
+		}
+
+		if (rules.maxValue !== undefined) {
```
Comment (nimietist) [inline fields/types/number/NumberValidation.js]: Do you think you should also check for nulls as well?

`rules.maxValue !== undefined || rules.maxValue !== null`

**Comment by nimietist** [comment]
👍 with the one comment

File: fields/types/number/NumberValidation.js (hunk 1 above)
Comment (WelltokCaleb) [inline fields/types/number/NumberValidation.js]: yes, but it should be && not ||, right?

Comment (nimietist) [inline fields/types/number/NumberValidation.js]: I need naps. Would `isNaN` be of proper use here?

**Pull request merged**